src/main/java/com/iwaconsolti/school/service/ServiceImpl.java

```diff
@@ -0,0 +1,146 @@
+package com.iwaconsolti.school.service;
+
+import com.iwaconsolti.school.model.Course;
+import com.iwaconsolti.school.model.Grade;
+import com.iwaconsolti.school.model.School;
+import com.iwaconsolti.school.model.Student;
+import com.iwaconsolti.school.repository.CourseRepository;
+import com.iwaconsolti.school.repository.GradeRepository;
+import com.iwaconsolti.school.repository.SchoolRepository;
+import com.iwaconsolti.school.repository.StudentRepository;
+import lombok.extern.java.Log;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@Service
+public class ServiceImpl implements SchoolService, StudentService, CourseService, GradeService {
```

Comment:
It is better to have a lass that implemente eah service, otherwise you are in risk of creating a huge class